app/retail/templates/emails/grants/new_supporter.html

```diff
@@ -1,94 +1,90 @@
-{% extends 'emails/template.html' %}
```

Comment:
Unnecessary trailing whitespace, vars need to use the input `subscription` and/or `grant` objects.  None of these variables will exist.